assets/src/components/Sidebar/CampaignThemeSidebar.js

```diff
@@ -109,59 +109,59 @@ export class CampaignThemeSidebar extends Component {
       if (isShallowEqual(this.state.meta, meta)) {
         return;
       }
-      this.setState({ meta });
+      this.setState({meta});
       savePreviewMeta();
       if (
         this.state.options === null
       ) {
         const options = await loadOptions(themeName);
-        this.setState({ options });
+        this.setState({options});
         if (themeName) {
           try {
-            const response = await fetch(`${ themeJsonUrl + themeName.replace('-new', '') }.json`);
+            const response = await fetch(`${themeJsonUrl + themeName.replace('-new', '')}.json`);
             const theme = await response.json();
             applyChangesToDom(theme, []);
           } catch (e) {
-            console.log(`Failed loading config for ${ themeName }`);
+            throw `Failed loading config for ${themeName}`;
```

Comment:
Maybe we should just disable the ESLint warning for `console.log` here instead? Not sure how `throw` behaves in comparison 🤔